org.eclipse.wildwebdeveloper.xml/plugin.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+   <extension-point id="XMLExtensions" name="%ExtPoint.XMLExtensions" schema="schema/XMLExtensions.exsd"/>
```

Comment:
Extension point ids usually start with lower case.
So it's be more something like `xmlExtensions`. And since it's really extension to the lsp4xml language server, it should probably be good to explicitly name it `lsp4xmlExtensions`

Reply:
Please dont use lsp4xml. It could be changes in xmlls but it is in discussion.

Reply:
Ok, let's use `xmllsExtension` then